force-app/main/default/classes/JoobleIntegration.cls

```diff
@@ -67,7 +78,8 @@ public with sharing class JoobleIntegration {
         String type;
         Datetime updated;
         String companyName;
-        String URL__ahhahahah_weblink;
```

Comment:
Boooooo Hisssssssss..... okay fine

Reply:
haha, for some reason it started complaining about invalid characters and when I changed it, it stopped complaining, otherwise I'd have left it